scripts/bb-dependencies.sh

```diff
@@ -211,7 +212,7 @@ RHEL*)
         libuuid-devel libblkid-devel libselinux-devel \
         xfsprogs-devel libattr-devel libacl-devel libudev-devel \
         device-mapper-devel openssl-devel libffi-devel libaio-devel \
-        pam-devel python-devel python-setuptools python-cffi
+        pam-devel pamtester python-devel python-setuptools python-cffi
```

Comment:
For RHEL you'll need to move this in to ` # Testing support libraries and tools` as well.